2021-10-07 16:26:44,178 INFO    MainThread:6058 [wandb_setup.py:_flush():69] setting env: {}
2021-10-07 16:26:44,179 INFO    MainThread:6058 [wandb_init.py:_log_setup():342] Logging user logs to output/wandb/offline-run-20211007_162644-25ut7c7f/logs/debug.log
2021-10-07 16:26:44,179 INFO    MainThread:6058 [wandb_init.py:_log_setup():343] Logging internal logs to output/wandb/offline-run-20211007_162644-25ut7c7f/logs/debug-internal.log
2021-10-07 16:26:44,179 INFO    MainThread:6058 [wandb_init.py:init():375] calling init triggers
2021-10-07 16:26:44,179 INFO    MainThread:6058 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'game': 'environment', 'mode': 'train', 'testmode': 'test_random', 'test_type': 'gan', 'library_size': 25, 'not_cuda': 0, 'netG': '', 'netD': '', 'manualSeed': 2059, 'out': 'output', 'input_dir': 'input', 'input_name': 'map_zero.txt', 'test_dir': 'test_maps', 'nfc': 64, 'ker_size': 3, 'num_layer': 3, 'full_map_size': 20, 'noise_update': 0.1, 'pad_with_noise': False, 'niter': 15, 'gamma': 0.1, 'lr_g': 0.0005, 'lr_d': 0.0005, 'beta1': 0.5, 'Gsteps': 1, 'Dsteps': 1, 'lambda_grad': 0.1, 'alpha': 100, 'token_insert': -2, 'add_prev': False, 'device': device(type='cpu'), 'nc_current': 3, 'out_': 'output/map_zero/', 'seed_road': None, 'token_list': ['-', 'W', 'X'], 'ImgGen': []}
2021-10-07 16:26:44,180 INFO    MainThread:6058 [wandb_init.py:init():424] starting backend
2021-10-07 16:26:44,180 INFO    MainThread:6058 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-07 16:26:44,199 INFO    MainThread:6058 [backend.py:ensure_launched():135] starting backend process...
2021-10-07 16:26:44,209 INFO    MainThread:6058 [backend.py:ensure_launched():140] started backend process with pid: 6069
2021-10-07 16:26:44,211 INFO    MainThread:6058 [wandb_init.py:init():429] backend started and connected
2021-10-07 16:26:44,223 INFO    MainThread:6058 [wandb_init.py:init():477] updated telemetry
2021-10-07 16:26:44,225 INFO    MainThread:6058 [wandb_init.py:init():540] starting run threads in backend
2021-10-07 16:26:46,752 WARNING MainThread:6058 [wandb_init.py:init():795] interrupted
Traceback (most recent call last):
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 775, in init
    run = wi.init()
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 543, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 801, in communicate_run_start
    result = self._communicate(rec)
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 539, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 77, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/av/anaconda3/envs/gpn/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
